server/Application.cfc

```diff
@@ -15,9 +15,9 @@ component {
 	this.mappings[ "/docs"     ] = this.baseDir & "docs";
 	this.mappings[ "/listener" ] = this.baseDir;
 	*/
-	this.assetBundleVersion = 35;  // see parent application.cfc
+	this.assetBundleVersion = 36;  // see parent application.cfc
```

Comment:
this is not needed?

Reply:
I thought I had already corrected these - sorry.
Done, now!